spyder/widgets/calltip.py

```diff
@@ -127,13 +127,6 @@ def show_tip(self, point, tip, cursor=None, completion_doc=None):
         """
         Attempts to show the specified tip at the current cursor location.
         """
-        # Don't attempt to show it if it's already visible and the text
-        # to be displayed is the same as the one displayed before.
-        if self.tip == tip:
```

Comment:
Since this line is going to be deleted maybe the change I did in PR #11361  creating a `reset_tooltip` function is not relevant any more?

Reply:
Yes, you are probably correct

Reply:
@goanpeca, please also address this comment so we can merge this one.

Reply:
Fixed!